bigbluebutton-tests/playwright/layouts/focusOnVideo.js

```diff
@@ -0,0 +1,34 @@
+const { expect } = require('@playwright/test');
+const Page = require('../core/page');
+const e = require('../core/elements');
+const { ELEMENT_WAIT_LONGER_TIME } = require('../core/constants');
+const { MultiUsers } = require('../user/multiusers');
+
+class FocusOnVideo extends MultiUsers {
+  constructor(browser, context) {
+    super(browser, context);
+  }
+
+  async test() {
+    await this.modPage.shareWebcam();
+    await this.modPage2.shareWebcam();
+    await this.modPage.waitAndClick(e.actions);
+    await this.modPage.waitAndClick(e.layoutSettingsModalButton);
+    await this.modPage.waitAndClick(e.focusOnVideo);
+    await this.modPage.waitAndClick(e.confirmButton);
+
+    const modPageCameraDockLocator = await this.modPage.getLocator(e.cameraDock);
+    await expect(this.modPage.page).toHaveScreenshot('moderator1-focus-on-video.png', {
+      maxDiffPixels: 1000,
+      mask: [modPageCameraDockLocator],
+    });
+
+    const modPage2CameraDockLocator = await this.modPage2.getLocator(e.cameraDock);
+    await expect(this.modPage2.page).toHaveScreenshot('moderator2-focus-on-video.png', {
+      maxDiffPixels: 1000,
```

Comment:
with maxDiffPixels set to `1000` occasionally I got an error, but it's just the icons' color and meeting name:
![image](https://user-images.githubusercontent.com/56703993/214656524-1036ba08-2791-406b-92dc-96d541c27969.png)

I'd say to use `maxDiffPixelRatio: 0.005` instead in all screenshots comparisons (increasing the pixels diff a little bit will work, but I still prefer pixelRatio)